aloha-core/src/test/java/com/eharmony/aloha/score/audit/take3/JavaTestTake3.java

```diff
@@ -0,0 +1,132 @@
+package com.eharmony.aloha.score.audit.take3;
+
+import com.eharmony.aloha.id.ModelId;
+import com.eharmony.aloha.id.ModelIdentity;
+import com.eharmony.aloha.score.audit.take3.EitherAuditor.EitherTC;
+import com.eharmony.aloha.score.audit.take3.EitherAuditor.Result;
+import com.eharmony.aloha.score.audit.take3.EitherAuditor.Success;
+import com.eharmony.aloha.score.audit.take3.OptionAuditor.OptionTC;
+import deaktator.reflect.runtime.manifest.ManifestParser;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.BlockJUnit4ClassRunner;
+import scala.Option;
+import scala.reflect.Manifest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+/**
+ * Shows how
+ * Created by ryan on 12/16/16.
+ */
+@RunWith(BlockJUnit4ClassRunner.class)
+public class JavaTestTake3 {
```

Comment:
This looks very nice.  Could you show an example instantiating from a JSON through the factory?